apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -100,6 +100,8 @@ type ScaledObjectSpec struct {
 	Triggers []ScaleTriggers `json:"triggers"`
 	// +optional
 	Fallback *Fallback `json:"fallback,omitempty"`
+	// +optional
+	InitialDelayCooldownPeriod int32 `json:"initialDelayCooldownPeriod,omitempty"`
```

Comment:
```suggestion
	InitialCooldownPeriod int32 `json:"initialCooldownPeriod,omitempty"`
```

And also please move this after CoolDown period in this struct.